src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,15 @@
+@mixin contentFlexCenterCenter() {
```

Comment:
```suggestion
@mixin contentFlexCenter {
```
you may remove brackets if mixin doesn't accept any argument

Reply:
Thank you for recommendations!